src/pybind/mgr/dashboard_v2/HACKING.rst

```diff
@@ -0,0 +1,451 @@
+Dashboard v2 Developer Documentation
+====================================
+
+The Python backend code of this module requires a number of Python modules to be
+installed. They are listed in file ``requirements.txt``.  Using `pip
+<https://pypi.python.org/pypi/pip>`_ you may install all required dependencies
+by issuing ``pip -r requirements.txt``.
```

Comment:
A hint in which directory this command must be executed would be helpful. Same to line 24 ``npm install``.

Reply:
Agreed - fixed.